### ISSUE TITLE
Applying attributes to table rows

### ISSUE THREAD

**Issue by cameronrdecker** [userfrosting/UserFrosting#787]
Tablesorter allows for some child row functionality, but it requires a class be added to the <tr> element of the row, which is currently hardcoded in the ufTable() function.

**Comment by alexweissman** [comment]
In general, we should probably make it possible to override the `_processAjax` (and other methods that `ufTable` implements).  `_processAjax` in particular, could be refactored into a few smaller methods as well.

**Comment by alexweissman** [comment]
@cameronrdecker check out `hotfix` ;-)

You can set a `rowTemplate` option now, and give it a callback or point it to a Handlebars script:

```
    <script id="user-table-row" type="text/x-handlebars-template">
        {{#ifx (calc rownum '%' 2) '==' 0}}
            <tr style="background-color: red">
        {{ else }}
            <tr>
        {{/ifx}}
    </script>
```

**Comment by alexweissman** [comment]
Implemented in 984decd7134582f9a37ecc588c4aa798cd1f3783 and 2c79af04f8178fa75f834d2ceff1e38069cac5f5, released in v4.1.13-alpha.  🎉
